packages/cli/src/zosjobs/list/jobs/Jobs.definition.ts

```diff
@@ -39,6 +39,12 @@ export const JobsDefinition: ICommandDefinition = {
                 "You can specify a wildcard according to the z/OSMF Jobs REST endpoint documentation, " +
                 "which is usually in the form \"JOB*\".",
             type: "string"
+        },
+        {
+            name: "execdata", aliases: ["ed"],
```

Comment:
Should we include a hyphen in the option name (`execdata` -> `exec-data`)? I don't see a specific option name requested by @MikeBauerCA in #1158, so I will defer to others reviewing this PR for their opinion. 🙂

Reply:
I changed it to 'exec-data'. 👍🏽

Reply:
I would vote for `exec-data` 😋 
Thanks for updating!